client/python/Dockerfile.redhat

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ARG BASE_IMAGE=registry.access.redhat.com/ubi8/ubi:8.4
+ARG FINAL_IMAGE=registry.access.redhat.com/ubi8/ubi-minimal:8.4
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
+FROM $BASE_IMAGE as base_build
+
+RUN groupadd --gid 5000 ovms && \
+    useradd --home-dir /home/ovms --create-home --uid 5000 --gid 5000 --shell /bin/bash --skel /dev/null ovms
+
+RUN dnf install -y unzip && \
+    curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v3.18.0/protoc-3.18.0-linux-x86_64.zip && \
+    unzip protoc-3.18.0-linux-x86_64.zip -d /opt/protoc
+
+ENV PATH=$PATH:/opt/protoc
+
+RUN mkdir /ovmsclient
+COPY lib /ovmsclient/lib
+
+RUN dnf install -y python38 make && \
+    cd /ovmsclient/lib && make build
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
+FROM $FINAL_IMAGE as final_build
+
+ENV OVMSCLIENT_DIR="/home/ovmsclient"
```

Comment:
do we need it as an env variable?
container can be started from different security context so it will not have access to home/ovmsclient folder

Reply:
We can use different directory for storing and using the client. 
Would you prefer something like: `/ovmsclient` ?